db/views/reporting_facility_state_daily_dimensions_v01.sql

```diff
@@ -0,0 +1,222 @@
+         UNION (select * FROM follow_up_blood_sugars)
+         UNION (select * FROM follow_up_prescription_drugs)
+         UNION (select * FROM follow_up_appointments)
+     ),
+     all_follow_ups_with_medical_histories AS (
+         SELECT DISTINCT ON (all_follow_ups.day_of_year, all_follow_ups.facility_id, all_follow_ups.patient_id)
+             all_follow_ups.patient_id,
+             all_follow_ups.patient_gender,
+             all_follow_ups.facility_id,
+             mh.diabetes,
+             mh.hypertension,
+             all_follow_ups.user_id,
+             all_follow_ups.visit_id,
+             all_follow_ups.visit_type,
+             all_follow_ups.visited_at,
+             all_follow_ups.day_of_year
+         FROM
+             all_follow_ups
+                 INNER JOIN medical_histories mh
+                            ON all_follow_ups.patient_id = mh.patient_id
+     ),
+
+     registered_patients_with_medical_histories AS (
+         SELECT DISTINCT ON (registered_patients.day_of_year, registered_patients.facility_id, registered_patients.patient_id)
+             registered_patients.patient_id,
+             registered_patients.patient_gender,
+             registered_patients.facility_id,
+             mh.diabetes,
+             mh.hypertension,
+             registered_patients.user_id,
+             registered_patients.visit_id,
+             registered_patients.visit_type,
+             registered_patients.visited_at,
+             registered_patients.day_of_year
+         FROM
+             registered_patients
+                 INNER JOIN medical_histories mh
+                            ON registered_patients.patient_id = mh.patient_id
+     ),
+
+     daily_registered_patients AS (
+         SELECT DISTINCT ON (facility_id, visit_date)
+             facility_id,
+             date(visited_at) as visit_date,
+             day_of_year,
+
+             count(*) AS daily_registrations_all,
+             count(*) FILTER (WHERE hypertension = 'yes') AS daily_registrations_htn_all,
+             count(*) FILTER (WHERE hypertension = 'yes' and patient_gender = 'female') AS daily_registrations_htn_female,
+             count(*) FILTER (WHERE hypertension = 'yes' and patient_gender = 'male') AS daily_registrations_htn_male,
+             count(*) FILTER (WHERE hypertension = 'yes' and patient_gender = 'transgender') AS daily_registrations_htn_transgender,
+             count(*) FILTER (WHERE diabetes = 'yes') AS daily_registrations_dm_all,
+             count(*) FILTER (WHERE diabetes = 'yes' and patient_gender = 'female') AS daily_registrations_dm_female,
+             count(*) FILTER (WHERE diabetes = 'yes' and patient_gender = 'male') AS daily_registrations_dm_male,
+             count(*) FILTER (WHERE diabetes = 'yes' and patient_gender = 'transgender') AS daily_registrations_dm_transgender,
+             count(*) FILTER (WHERE hypertension = 'yes' and diabetes = 'yes') AS daily_registrations_htn_and_dm_only_all,
```

Comment:
Do we need the "only" in the column name? Can we remove it?

Reply:
yeah, we could remove the only from the column name